receiver/mysqlreceiver/documentation.md

```diff
@@ -643,6 +643,40 @@ The number of hits, misses or overflows for open tables cache lookups.
 | ---- | ----------- | ------ |
 | status | The status of cache access. | Str: ``hit``, ``miss``, ``overflow`` |
 
+## Default Events
+
+The following events are emitted by default. Each of them can be disabled by applying the following configuration:
+
+```yaml
+events:
+  <event_name>:
+    enabled: false
+```
+
+### db.server.query_sample
+
+query sample event
+
+#### Attributes
+
+| Name | Description | Values |
+| ---- | ----------- | ------ |
+| db.system.name | The name of the database system. | Str: ``mysql`` |
+| mysql.thread_id | The unique identifier for the thread executing the statement. | Any Int |
+| user.name | The user associated with a foreground thread, empty for a background thread (originally processlist_user). | Any Str |
+| db.namespace | The default database for the thread, or empty if none has been selected (originally processlist_db). | Any Str |
+| mysql.processlist_command | The type of command the thread is executing on behalf of the client for foreground threads, or `Sleep` if the session is idle. | Any Str |
+| mysql.processlist_state | An action, event, or state that indicates what the thread is doing. | Any Str |
+| mysql.sql_text | The SQL statement text for the event. | Any Str |
+| mysql.digest | The statement digest SHA-256 value as a string of 64 hexadecimal characters, or empty if the statements_digest consumer is no. | Any Str |
+| mysql.event_id | The thread associated with the event and the thread current event number when the event starts. | Any Int |
+| mysql.wait_event | The name of the instrument that produced the event. | Any Str |
```

Comment:
Since it is an inferred field, which means there is no direct field mapped to it, should we call it `mysql.wait_type` to be consistent with the SQL Server receiver? https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/f945dfd21ae8353569682278b17ba8f373dd8367/receiver/sqlserverreceiver/logs-documentation.md?plain=1#L34

Reply:
I agree, we can use mysql.wait_type